Database/Corrections/SeasonOfDiscovery.lua

```diff
@@ -284,6 +285,7 @@ local questsToBlacklistBySoDPhase = {
         [90145] = true, -- Hiding Druid Lacerate Darkshore for now as there are too many icons
         [90146] = true, -- Hiding Druid Mangle Mulgore for now as there are too many icons
         [90147] = true, -- Hiding Paladin Hand of Reckoning Westfall for now as there are too many icons
+        [90148] = true, -- Hiding Druid Sunfire for now as there are too many icons
```

Comment:
Instead of hiding this, we should cleanup the Mulgore spawns of the "Lunar Stone" NPC (207577)

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5457/commits/3cdd3bb1b35173d83f9b608c62aa760dd301d2cb